.rubocop.yml

```
@@ -16,9 +16,9 @@ AllCops:
   TargetRubyVersion: 3.0
 
   Exclude:
-    - tmp/development_apps/**/*
-    - tmp/test_apps/**/*
-    - gemfiles/rails_61/vendor/bundle/**/*
+    - gemfiles/**/vendor/bundle/**/*
```

We should also include the `.git` directory, since we override the `Exclude` config we have to redeclare it as the defaults aren't merged with our custom setting. https://github.com/rubocop/rubocop/issues/9325

```suggestion
    - .git/**/*
    - .github/**/*
    - bin/**/*
    - gemfiles/**/vendor/bundle/**/*
```

---

Thanks, good catch.

I've force pushed with some changes (just adding the defaults + customs, so `vendor/` instead of `vendor/bundle`

The performance is also improved

---

Ok great, thank you. I've had to use the same config at work previously after uncovering that issue. All the changes look good and nice to see the performance improvement.